test/networkd-test.py

```diff
@@ -49,6 +49,14 @@
 @unittest.skipIf(networkd_active,
                  'networkd is already active')
 class ClientTestBase:
+    @classmethod
+    def setUpClass(klass):
+        subprocess.check_call(['systemd-analyze', 'set-log-level', 'debug'])
+
+    @classmethod
+    def tearDownClass(klass):
+        subprocess.check_call(['systemd-analyze', 'set-log-level', 'info'])
```

Comment:
> I didn't see a way to query the current log level, do you know one?

`systemctl show --property LogLevel --value --no-pager`

Anyway, this PR looks good to me